org.bzflag.BZFlag.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "org.bzflag.BZFlag",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "bzfs",
```

Comment:
Is `command` the executable that will be running when an icon is double clicked? If so, this should be the `bzflag` executable. `bzfs` is the server component of the game for hosting game servers.

Reply:
I believe you are correct there.